GridList/GridList.jsx

```diff
@@ -0,0 +1,178 @@
+      classes.push("mdc-grid-list--tile-gutter-1");
+    }
+
+    if (props[this._propsDict.secondaryActionIconAlign]) {
+      classes.push(
+        "mdc-grid-list--with-icon-align-" +
+          props[this._propsDict.secondaryActionIconAlign]
+      );
+    }
+
+    if (props[this._propsDict.aspectRatio]) {
+      classes.push(
+        "mdc-grid-list--tile-aspect-" + props[this._propsDict.aspectRatio]
+      );
+    }
+
+    if (props[this._propsDict.twoLineCaptions]) {
+      classes.push("mdc-grid-list--twoline-caption");
+    }
+
+    return classes.join(" ");
+  }
+  materialDom(props) {
+    const cols = props.cols || 2;
+    const tileWidthPct = Math.floor(100 / cols) + "%";
+
+    return (
+      <span>
+        <style
+          dangerouslySetInnerHTML={{
+            __html: `
+        .mdc-grid-list__tiles {
+          --mdc-grid-list-tile-width: ${tileWidthPct};
+        }
+        .mdc-grid-list__tiles > .mdc-grid-tile > * img {
+          width: 100%;
+        }
+      `
+          }}
+        />
+        <div
+          className={this.createClassName(props)}
+          ref={control => {
+            this.control = control;
+          }}
+          {...props}
+        >
+          <ul class="mdc-grid-list__tiles">{props.children}</ul>
+        </div>
+      </span>
+    );
+  }
+}
+
+/**
+ * @prop actionIcon	{Node} - An IconButton element to be used as secondary action target (primary action target is the tile itself).
+ * @prop actionPosition {'left'|'right'} - Position of secondary action IconButton.
+ * @prop children	{Node} - Theoretically you can pass any node as children, but the main use case is to pass an img, in whichcase GridTile takes care of making the image "cover" available space (similar to background-size: cover or to object-fit:cover).
+ * @prop cols	{number} - Width of the tile in number of grid cells.
+ * @prop containerElement	{string|node} - Either a string used as tag name for the tile root element, or a ReactElement. This is useful when you have, for example, a custom implementation of a navigation link (that knows about your routes) and you want to use it as the primary tile action. In case you pass a ReactElement, please ensure that it passes all props, accepts styles overrides and render it's children.
+ * @prop rows	{number} - Height of the tile in number of grid cells.
+ * @prop style {object} - Override the inline-styles of the root element.
+ * @prop subtitle	{Node} - String or element serving as subtitle (support text).
+ * @prop subtitleStyle {object} - Override the inline-styles of the subtitle element.
+ * @prop title {Node} - Title to be displayed on tile.
+ * @prop titleBackground {string} - 'rgba(0, 0, 0, 0.4)'	Style used for title bar background. Useful for setting custom gradients for example
+ * @prop titlePosition {'top'|'bottom'} - Position of the title bar (container of title, subtitle and action icon).
+ * @prop titleStyle {object} - Override the inline-styles of the title element.
```

Comment:
where are these used?

Reply:
nowhere yet. they are stubs taken from http://www.material-ui.com/#/components/grid-list under the assumption we'd want to be compatible with that api